SysTrayIcon/explainer.md

```diff
@@ -0,0 +1,149 @@
+
+### System *Status* area
+
+Instead of menu bar extra or system tray area, we use the term "*System Status area*" to refer to the OS surface where the platform displays information like network, time, battery, etc. Most major platforms have a similar surface, and they can also have mechanisms that allow apps to participate in this surface. Web apps may have a presence in the "system status" area, and present certain functionality to end users. 
+
+For example, on Windows, this is the area in the bottom-right corner of the desktop. The screenshot below shows a file backup app, a videoplayer, an email client and an instant messaging program running and having presence in the *system tray* area.
+![Image of Windows System Status](SystrayWindows.png)
+
+In macOS, this is the area in the top-right corner of the desktop. In the screenshot below we can see a messaging app, an email client, a VPN client, a videoplayer and a file backup program running and having an icon in their menu bar area. This is called "[menu bar extras](https://developer.apple.com/design/human-interface-guidelines/the-menu-bar#Menu-bar-extras)".
+
+![Image of macOS Menu Bar](MacMenuBar1.png)
+
+Other platforms like Ubuntu, that use the GNOME GUI can be tweaked to display apps on the "GNOME shell top bar", altohugh this requires the use of [GNOME extensions](https://extensions.gnome.org/extension/1160/dash-to-panel/). Ubuntu does not support this functionality out of the box, so it is out of the scope of this feature to cover this platform.
+
+Having icons on the system status area for apps is giving the user an entry point into the application, potentially displaying additional useful information at a glance or providing menus to deep-link into the app. Some of the benefits for the user in the previous screenshots are:
+* Quick re-entry into every app.
+* Knowing if an app has notifications, or the online state of a instant messagaging client.
+* Knowing at a glance if the VPN is active or not.
+* Quitting the app through the menu.
+* Adapting to the expected paradigm of running desktop apps that have these icons.
+
+## Goals
+* Enable an installed Web App to have an icon on the "System Status" area.
+* Enable Web Applications to have a menu associated with the icon in the System Status area.
+* Enable badging for System Status Icons.
+
+## Non-goals
+* Allow a Web Application to have a custom UX in the System Status menu.
+    * The minimum common UX that major platforms offer is a menu. It is not the scope of this API to enable elaborate menus that different platforms may offer.
+* Allow the app to run on the background.
+    * This API is to allow an installed web app to have an icon on the system status area when the app is open/running.
+* Create dynamic menus that change options.
```

Comment:
I think we _do_ need to be able to dynamically change the status icon though.  Like if a messaging app wants the icon to represent your current status, or if an app wants to indicate if it is/isn't actively connected to the service.